types/parse/index.d.ts

```diff
@@ -254,10 +254,19 @@ declare namespace Parse {
      *     extension.
      */
     class File {
-        constructor(name: string, data: any, type?: string);
+        constructor(name: string, data: number[] | { base64: string } | Blob | { uri: string }, type?: string);
```

Comment:
Hmm, this breaks when using Parse on the server side (`import Parse = require('parse/node')`) since Node.js doesn't have a `Blob` defined globally (it's a browser global).

Not sure how to best fix this though, any ideas @RaschidJFR @rbuckton? 🤔

Reply:
Thanks for pointing that out. I've changed it in #40990
That should do. What do you think @LinusU ?